modules/test/test-hello-world.bats

```diff
@@ -19,3 +19,13 @@ load 'test/test_helper.bash'
     assert_success
     assert_output --partial "RESULT: bar"
 }
+
+@test "[hello-world] - KvHelloWorldScoped.java" {
```

Comment:
ah, I reordered some test examples in Howtos yesterday, as I was expecting them to be in Alphabetical order.
I see here that's not the case 😄